### PR TITLE
chore(99-general-concepts): agrega presentación de la unidad 01-debugging y detalles finales

### PR DESCRIPTION
**Pull request by ivandevp**
Últimos detalles del curso @nicolethenerd ! 🎉 👍 💪

### CODE REVIEW

File: 99-general-concepts/README.md [hunk 1]
```diff
@@ -1,107 +1,54 @@
...
-* A entender las diferencias y entre el estilo **imperativo vs declarativo**
-* Las diferencias entre **constructores** y **factories**
-* Cómo implementar **herencia prototipal**
-* Los principios básicos de la **programación funcional**
-* A usar **composición** como alternativa la _herencia_
-* A **trabajar sobre una base de código existente**
-* A trabajar haciendo **pair programming**
-* A sacarle el jugo a git y GitHub usando **issues**, **milestones**,
-  **branches**, **forks** y **pull requests**
+* Cómo reaccionar ante mensajes de errores comunes
```
Comment (nicolethenerd) [inline 99-general-concepts/README.md]: comunes -> comúnes

File: 99-general-concepts/README.md [hunk 2]
```diff
@@ -1,107 +1,54 @@
...
-| 02 | práctica | self-paced | Xmin | [Ejercicios](02-tdd/02-self-learning-2.md)
-| 03 | quiz | self-paced | Xmin | [Cuestionario](02-tdd/03-some-quiz.md)
-| 04 | seminario | guiado | Xmin | [Ejercicios guiados](02-tdd/04-ejercicios-guiados.md)
-| 05 | seminario | guiado | Xmin | [Solucionario](02-tdd/05-solucionario.md)
-| 06 | quiz | self-paced | Xmin | [Requiz](02-tdd/06-requiz.md)
-| 07 | lectura | self-paced | Xmin | [Cierre](02-tdd/07-cierre.md)
+| 00 | lectura | slef-paced | 10min | [Apertura](01-debugging/00-apertura.md)
+| 01 | lectura | self-paced | 30min | [¿Qué es debugging?](01-debugging/01-self-learning-1.md)
+| 02 | lectura | self-paced | 60min | [Mensajes de Error](01-debugging/02-self-learning-2.md)
+| 03 | práctica | self-paced | 60min | [Retos de código](01-debugging/03-some-quiz.md)
```
Comment (nicolethenerd) [inline 99-general-concepts/README.md]: 03-code-challenges.md

File: 99-general-concepts/README.md [hunk 3]
```diff
@@ -1,107 +1,54 @@
...
+| 04 | lectura | slef-paced | 40min | [¿Cómo debuggear?](01-debugging/04-ejercicios-slef-paceds.md)
```
Comment (nicolethenerd) [inline 99-general-concepts/README.md]: 04-breakpoints.md

slef-paced -> self-paced

File: 99-general-concepts/README.md [hunk 4]
```diff
@@ -1,107 +1,54 @@
...
+| 05 | lectura | slef-paced | 10min | [Cierre](01-debugging/07-cierre.md)
```
Comment (nicolethenerd) [inline 99-general-concepts/README.md]: slef-paced -> self-paced

05-closing.md

File: 99-general-concepts/README.md (hunk 1 above)
Comment (ivandevp) [reply 99-general-concepts/README.md]: It's just _comunes_

**Comment by ivandevp** [comment]
@nicolethenerd I made the changes 😃

**Pull request merged**